### ISSUE TITLE
Chromium only tries to redownload the local file, won't install it.

### ISSUE THREAD

**Issue by mmmmna** [cherdt/they-live#1]
Chromium: Version 79.0.3945.79 (Official Build) Built on Ubuntu , running on Ubuntu 18.04 (64-bit)

I'm using Mate 1.20.1-2ubuntu1  and Caja 1.20.2 is the file manager.

I've downloaded the theylive.crx, checked MD5, all is good. When I drag the local file into chrome://extensions/, chrome only offers to redownload the file. Developer mode on and off. no different.

![Screenshot at 2020-01-10 16-12-54](https://user-images.githubusercontent.com/31926804/72187057-a864b400-33c4-11ea-9ad1-a87d1431bbee.png)

**Comment by cherdt** [comment]
I was just able to install this on a fresh Chromium install:

    Version 80.0.3987.87 (Official Build) Built on Ubuntu , running on Ubuntu 18.04 (64-bit)

I confirmed that it works with the example page listed in the README.

I was unable to download the file via Chromium (it gave me the error `CRX_REQUIRD_PROOF_MISSING`) but I was able to download it via another method (Firefox, curl, wget, etc.) and use drop-and-drag.

Are you able to download the file at all? From your screenshot I could not tell that the file exists on your filesystem. If downloading the file is the problem, try this from your terminal:

    curl -o ~/Downloads/theylive.crx https://osric.com/chris/steganography/theylive.crx

**Comment by mmmmna** [comment]
Good now. Murphy lives here, I guess. I used the curl link method in the prior message, the file downloaded, and THIS time, Chrome installed it... but the MD5 for the first effort matched what it was supposed to match? Of course, it wouldn't be the first time in my life that an invalid file was served and the MD5 checksum was for the invalid file.

curl file md5sum after it arrived here: fb7db8fca9b70777424742d1aa79c437  theylive.crx
I've deleted the former crx file because it was not functional.

Thanks for the response.
